Assets/_Game/Scripts/Managers/GameSystems.cs

```diff
@@ -37,6 +37,11 @@ private void InitSystems()
         LevelSceneManager = new LevelSceneManager();
 
         // Start the game
-        StartCoroutine(StateManager.NavigateToState(typeof(MainMenuState)));
+        StateManager.NavigateToState(typeof(MainMenuState));
+    }
+
+    public void StartChildCoroutine(IEnumerator coroutineMethod)
```

Comment:
You don't need to make a separate method. Gamesystems is a monobehavior so for systems outside of itself (IE StateManager) you can just use its reference and call .StartCoroutine as you would normally. I'll show below in StateManager